packages/components/src/components/navigation-item/navigation-item.scss

```
@@ -104,15 +114,19 @@
 		}
 	}
 
-	&[aria-current="page"] {
-		font-weight: 700;
-	}
-
+	&:not(:has([aria-current="page"])),
 	&:not([aria-current="page"]) {
 		// revert for sub-navigation
 		font-weight: normal;
 	}
 
+	&:has([aria-current="page"]),
+	&[aria-current="page"] {
+		// add puls for main-navigation items
+		@extend %show-db-puls-auto;
+		font-weight: 700;
+	}
```

the related text, that you're trying to set as `font-weight: 700;` is always included as a hyperlink, isn't it?

This could simplify the declaration:
```suggestion
	&:has([aria-current="page"]),
	&[aria-current="page"] {
		// add puls for main-navigation items
		@extend %show-db-puls-auto;
		& > a {
			font-weight: 700;
		}
	}
```